apps/charterafrica/src/components/Datasets/index.js

```diff
@@ -1,3 +1,5 @@
+import Dataset from "./Dataset";
```

Comment:
Isn't `Dataset` a standalone component? i.e. Why does it need to be a child of `Datasets`?